lib/optimizely/decision_service.rb

```diff
@@ -387,13 +357,26 @@ def save_user_profile(user_profile, experiment_id, variation_id)
       user_id = user_profile[:user_id]
       begin
         user_profile[:experiment_bucket_map][experiment_id] = {
-          :variation_id => variation_id
+          variation_id: variation_id
         }
         @user_profile_service.save(user_profile)
         @config.logger.log(Logger::INFO, "Saved variation ID #{variation_id} of experiment ID #{experiment_id} for user '#{user_id}'.")
       rescue => e
         @config.logger.log(Logger::ERROR, "Error while saving user profile for user ID '#{user_id}': #{e}.")
       end
     end
+    def get_bucketing_id_from_attributes(user_id, attributes)
```

Comment:
please rename this method to get_bucketing_id.

Reply:
Updated.